src/main/java/org/folio/rest/impl/PatronNoticeResourceImpl.java

```diff
@@ -56,10 +57,10 @@ public void postPatronNotice(String lang, PatronNoticeEntity entity,
       });
   }
 
-  OkapiModulesClient makeOkapiModulesClient(Context vertxContext, Map<String,
+  NoticesClient getNoticesClient(Context vertxContext, Map<String,
```

Comment:
Let's leave the "make" prefix. This method creates new instance of a client but "get" makes it look like a getter.

Reply:
edited